package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "fhir-works-on-aws-routing",
+  "name": "@abacus-insights/fhir-works-on-aws-routing",
```

Comment:
Do we need to update the dependencies below to point to `@abacus-insights/fhir-works-on-aws-interface`?

Reply:
How else does publish know where to push to?